content/200-concepts/100-components/01-prisma-schema/03-generators.mdx

```diff
@@ -6,38 +6,41 @@ metaDescription: 'Generators in your Prisma schema specify what assets are gener
 
 <TopBlock>
 
-A generator determines which assets are created when you run the `prisma generate` command, including:
+A Prisma schema can have one or more generators, represented by the [`generator`](../../../reference/api-reference/prisma-schema-reference#generator) <span class="api"></span> block:
 
-- The **Prisma client (language specific)** (currently only `prisma-client-js` is available)
+```prisma
+generator client {
+  provider = "prisma-client-js"
+  output   = "./generated/prisma-client-js"
+}
+```
+
+A generator determines which assets are created when you run the `prisma generate` command. The main property `provider` defines which **Prisma Client (language specific)** is created - currently, only `prisma-client-js` is available. Alterantively you can define any Npm package that follows our generator specification. Additionally and optionally you can define a custom output folder for the generated assets with `output`.
+
+</TopBlock>
 
-  - Target **engine binaries** for `prisma-client-js` (for example, `debian-openssl-1.0.x` if you are deploying to Ubuntu 14.04, or `native` if you are working locally)
+## `prisma-client-js`
 
-- Preview features to include
-- A [custom output folder](../prisma-client/working-with-prismaclient/generating-prisma-client#using-a-custom-output-path) for the generated assets
+The generator for Prisma's Javascript Client accepts multiple additional properties:
 
-A Prisma schema can have one or more generators, represented by the [`generator`](../../../reference/api-reference/prisma-schema-reference#generator) <span class="api"></span> block. The following example specifies a `provider` and a list of `previewFeatures`:
+- `binaryTargets`: Target **engine binaries** for `prisma-client-js` (for example, `debian-openssl-1.0.x` if you are deploying to Ubuntu 14.04, or `native` if you are working locally)
+- `previewFeatures`: Preview features to include
```

Comment:
```suggestion
- `previewFeatures`: [Preview features](../preview-features) to include
```

Reply:
Nope, that is a _great_ suggestion actually.